irods/irods_runtime_init.yml

```diff
@@ -151,3 +151,18 @@
         stdin: "{{ _irods_admin_password }}"
       register: results
       changed_when: results.stdout == 'scheduled trash removal'
+  
+    - name: Ensure that the starting collections get UUIDs
+      delegate_to: localhost
+      become: false
```

Comment:
The task should be executed on an irods_catalog host instead of the control node. From our Austrian deployment, I just learned that we can't assume iCommands, e.g., `imeta` are installed on the control node. The previous tasks calling `irule` on the control node were changed earlier today to run on a managed node.